README.md

```diff
@@ -1,5 +1,9 @@
 # Kubernetes Enterprise Integration Patterns (Keip)
 
+[![operator-workflow](https://github.com/OctoConsulting/keip/actions/workflows/operator.yml/badge.svg?branch=main)](https://github.com/OctoConsulting/keip/actions/workflows/operator.yml)
+[![webhook-workflow](https://github.com/OctoConsulting/keip/actions/workflows/webhook.yml/badge.svg?branch=main)](https://github.com/OctoConsulting/keip/actions/workflows/webhook.yml)
+[![minimal-app-workflow](https://github.com/OctoConsulting/keip/actions/workflows/minimal-app.yml/badge.svg?branch=main)](https://github.com/OctoConsulting/keip/actions/workflows/minimal-app.yml)
```

Comment:
:memo: nitpick, but it might look bit cleaner if you removed the `workflow` suffix from each of the labels.